docs/projects/cn0577/index.rst

```diff
@@ -0,0 +1,294 @@
+.. _cn0577:
+
+CN0577 HDL project
+================================================================================
+
+Overview
+-------------------------------------------------------------------------------
+
+The :adi:`EVALCN0577-FMCZ <CN0577>` provides an analog front-end and an FMC
```

Comment:
```suggestion
The :adi:`CN0577` provides an analog front-end and an FMC
```
It's enough to be referred just like this. Should be updated in the whole document